src/qibo_cloud_backends/aws_client.py

```diff
@@ -0,0 +1,141 @@
+from qibo.backends import NumpyBackend
+from qibo.config import raise_error
+from qibo.result import MeasurementOutcomes, QuantumState
+
+import re
+import importlib.util
+import sys
+
+# Import Qiskit packages for transpiler
+from qiskit import transpile
+from qiskit import QuantumCircuit
+from qiskit.transpiler import CouplingMap
+from qiskit.circuit.random import random_circuit
+from qiskit import qasm2
+
+from braket.aws import AwsDevice, AwsQuantumTask
+from braket.circuits import Gate, observables
+from braket.circuits import Circuit as braket_Circuit
+# from braket.device_schema import DeviceActionType
+from braket.devices import Devices, LocalSimulator
+# from braket.parametric import FreeParameter
+# from braket.ir.openqasm import Program
+
+class AWSClientBackend(NumpyBackend):
+    """Backend for the remote execution of AWS circuits on the AWS backends.
+
+    Args:
+        device_arn (str | None): The ARN of the Braket device.
+            If `None`, instantiates the `LocalSimulator("default")`.
+    """
+
+    def __init__(self, device=None):
+        super().__init__()
+
+        if device is None:
+            self.device = LocalSimulator("default")
+            # self.device = LocalSimulator("braket_dm")
+        else:
+            self.device = device
+        self.name = "aws"
+
+    def remove_qelib1_inc(self, qasm_string):
+        # Remove the "include "qelib1.inc";\n" line
+        modified_code = re.sub(r'include\s+"qelib1.inc";\n', '', qasm_string)
+        return modified_code
+
+    def qasm_convert_gates(self, qasm_code):
+        # To replace the notation for certain gates in OpenQASM
+        # To add more gates if necessary
+        dict = {
```

Comment:
Instead of defining dictionary each time the method is called, make it a constant at the top of the file:

```
_QASM_BRAKET_GATES = {
    "id": "i",
    ...
}

class BraketBackend(NumpyBackend):
    ...
```